### DISCUSSION TITLE
Agree on solution for storing image assets source files

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#94] [Content] [answered]
Quoting Chris Mills, who started an issue around this originally:

> One of our contributors mentioned that it'd be really good to have a place to store image source files, e.g. the raw SVG or draw.io source version of the final image shown on an article, so that if it needs to be edited later on, the source version can be easily found. Could we just start including an `img-source` folder in the Yari article folders? Would be good to discuss this at some point.

More details here: https://github.com/mdn/yari/issues/1678

**Comment by hamishwillee** [comment]
I'm OK with the idea of having sources for images in a known-named folder. Perhaps in particular if we also had a naming convention for the files - i.e. the name-part must match the generated format. For files that are generated on draw.io (or whatever) you could have a text file with a link to the proper file.

**Comment by teoli2003** [comment]
I really like the idea of a folder with a specific name (so that it is not available through `https://developer.mozilla.org`: I have lost so many source files over the year (stored locally, then the computer had to be wiped out…)

I see in the original issues that the name `_img-source` has been proposed. I have no strong feelings here, but I would even settle for something simpler like `_sources`.

**Reply by hamishwillee** [reply]
I'd prefer `_img-source` if this is just for images. Even the index.md is a source so `_sources` might require more clarification for users to infer the purpose.

**Comment by bsmth** [comment, marked as answer]
Currently we're starting to put source files (and output images / SVG if you want to serve them from GH pages) for diagrams in here: https://github.com/mdn/shared-assets/tree/main/images/diagrams
